packages/contracts/contracts/voting/addresslist/Addresslist.sol

```diff
@@ -0,0 +1,103 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.8.10;
+
+import {CheckpointsUpgradeable} from "@openzeppelin/contracts-upgradeable/utils/CheckpointsUpgradeable.sol";
+
+import {_uncheckedAdd, _uncheckedSub} from "../../utils/UncheckedMath.sol";
+import {TimeHelpers} from "../../utils/TimeHelpers.sol";
+
+/// @title Addresslist
+/// @author Aragon Association - 2021-2022.
+/// @notice The majority voting implementation using an list of member addresses.
+/// @dev This contract inherits from `MajorityVotingBase` and implements the `IMajorityVoting` interface.
+abstract contract Addresslist is TimeHelpers {
```

Comment:
Yes, it's needed.. Later on, we will do free-level functions instead of the `Timehelpers` contract. Though, your comment was helpful, `AddressList` doesn't need it, but inheritting contrcats do need it. 

see https://github.com/aragon/core/pull/167/commits/44f8dae72a81ec807668682e96fa0e4c4f9b31e2